README.md

```diff
@@ -22,7 +22,7 @@ Here's what a production database.yml file might look like:
 production:
   adapter:            postgresql
   encoding:           unicode
-  host:               <%= Diplomat::Service.get('postgres').Address %>
+  host:               <%= Diplomat::Service.get('postgres').ServiceAddress %>
```

Comment:
Most of the time ServiceAddress can be `""`, in that case, we should do:
```
service_instance = Diplomat::Service.get('postgres').ServiceAddress
service_instance.ServiceAddress || service_instance.Address
```
I suspect no ?